test/unit/pr_checker.test.js

```diff
@@ -474,11 +489,40 @@ describe('PRChecker', () => {
         reviews: approvingReviews,
         commits: simpleCommits,
         collaborators
-      });
+      }, argv);
 
       const status = checker.checkCommitsAfterReview();
       assert.deepStrictEqual(status, true);
     });
+
+    it('should print out three commits if passed NaN', () => {
```

Comment:
Shouldn't we test cases like `warnCommits: 0`?

Reply:
Also I am kinda curious if `warnCommits: -1`...should be `slice(1)`, which kinda make sense

Reply:
`maxCommits: -1` Would warn first commit after review and so on. `0` would warn all commits! which it should not. Here is what i am thinking...

```javascript
maxCommits = -maxCommits;
// In this case if negative number or zero was passed
// it should not warn out commits. mainly because it seems pretty 
// buggy to log first commits in case of negatives!
maxCommits = maxCommits > -1 ? commits.length : maxCommits;  
```

Reply:
@joyeecheung updated as follows. if `0` is passed it should not log anything. if `-1` or any negative number convert it to positive, and if `NaN` in case of string is passed it shoule default to `3`.